net.shadps4.shadPS4.yaml

```diff
@@ -78,8 +77,9 @@ modules:
       # don't include submodules, which shadps4 depends on (in the externals directory)
       - type: git
         url: https://github.com/shadps4-emu/shadPS4
-        commit: fc50567fc2e5be7f061f22981e6f98082bc9fe27
+        commit: d94abffd9a6bba013fe2524a7e085ccfa253ced9
+        tag: Pre-release-shadPS4-2025-01-13-d94abff
         x-checker-data:
           type: json
-          url: https://api.github.com/repos/shadps4-emu/shadps4/commits/main
-          commit-query: .sha
+          url: https://api.github.com/repos/shadps4-emu/shadps4/releases
+          tag-query: map(select(.prerelease)) | first | .tag_name
```

Comment:
I see a single pre-release tag. how often are these supposed to be created?

Reply:
I think the pre-releases get updated too frequently to be compliant with the flathub policy.  The emulator is still very new and doesnt have as mature of a release cadence as your average project.

Until there is a more clear distinction between nightly and beta, we should drop the beta branch.

Reply:
What should we do to remove the beta channel?

Reply:
you can't remove it, since you published to it once, just let it be.

Reply:
Thanks. I will disable it in CI too.